pallets/marketplace/src/lib.rs

```diff
@@ -41,19 +46,26 @@ pub mod pallet {
     use crate::RewardManager;
 
     #[pallet::config]
-    pub trait Config: frame_system::Config + pallet_acurast::Config {
+    pub trait Config:
+        frame_system::Config + pallet_acurast::Config + pallet_timestamp::Config
```

Comment:
@gitsimon why do we need a strong coupling with `pallet_timestamp::Config` instead of doing it line in `pallet_acurast`?